csharp-intermediate-topics/DifferencesBetweenDynamicTypes/DifferencesBetweenDynamicTypes/Person.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Dynamic;
+
+namespace DifferencesBetweenDynamicTypes;
+
+public class Person : DynamicObject
+{
+    private Dictionary<string, object> personalInformation;
+
+    public Person()
+    {
+        personalInformation = new Dictionary<string, object>();
+    }
+
+    public override bool TryGetMember(GetMemberBinder binder, out object result)
```

Comment:
This should accept returning null values as well
```suggestion
    public override bool TryGetMember(GetMemberBinder binder, out object? result)
```